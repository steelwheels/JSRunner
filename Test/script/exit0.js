
function main(args)
{
	console.log("Before exit: " + args) ;
	exit(0) ;
	console.log("After exit") ;
}

