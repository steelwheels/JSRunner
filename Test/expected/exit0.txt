Before exit: a,b,c
